### PR TITLE
fix module onlyread-sql

### PR DESCRIPTION
**Pull request by piensaenpixel**
@nobuti I suppose this is yours, could you take a look?

Fix -  https://github.com/CartoDB/cartodb/issues/8176

image

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/components/code-mirror/code-mirror-view.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ module.exports = CoreView.extend({
       theme: 'material',
       mode: 'cartocss',
       scrollbarStyle: 'simple',
-      lineWrapping: true,
+      lineWrapping: 'true',
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/editor/components/code-mirror/code-mirror-view.js]: This is a bit weird, why as string? I can see it's working as boolean.

**Comment by nobuti** [comment]
I've declared the infobox as code in the view and everything works like a charm 💯

File: lib/assets/javascripts/cartodb3/editor/components/code-mirror/code-mirror-view.js (hunk 1 above)
Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/editor/components/code-mirror/code-mirror-view.js]: yeah forget it, i'll fix it

**Comment by xavijam** [comment]
❤️

**Comment by matallo** [comment]
`package.json`?

**Comment by xavijam** [comment]
true @matallo, thanks. So if we change package, we need to update shrinkwrap :S.

**Comment by piensaenpixel** [comment]
i dont know what you mean ... sorry

**Comment by xavijam** [comment]
bump the package.json version, and then run the `npm run intern...`

**Comment by matallo** [comment]
current `package.json`is already "3.27.38" https://github.com/CartoDB/cartodb/blob/master/package.json#L3

**Comment by piensaenpixel** [comment]
fixed

**Comment by xavijam** [comment]
Conflicts

**Comment by piensaenpixel** [comment]
resolved

**Comment by xavijam** [comment]
:+1:, let's wait for the frontend tests and then go

**Comment by xavijam** [comment]
merge!

**Pull request merged**